runtime/doc/eval.txt

```diff
@@ -3621,9 +3780,12 @@ diff_hlID({lnum}, {col})				*diff_hlID()*
 
 empty({expr})						*empty()*
 		Return the Number 1 if {expr} is empty, zero otherwise.
-		A |List| or |Dictionary| is empty when it does not have any
-		items.  A Number is empty when its value is zero.  Special
-		variable is empty when it is |v:false| or |v:null|.
+		- A |List| or |Dictionary| is empty when it does not have any
+		  items.
```

Comment:
```diff
+               - A |String| is empty when its length is zero.
```
is missing. Sync with Vim's doc for `empty()`, excluding unsupported types (ie. job).